test/hexpm/web/controllers/dashboard_controller_test.exs

```diff
@@ -139,6 +139,15 @@ defmodule Hexpm.Web.DashboardControllerTest do
     assert_delivered_email(Hexpm.Emails.password_changed(c.user))
   end
 
+  test "Show user API keys", c do
+    conn =
+      build_conn()
+      |> test_login(c.user)
+      |> get("dashboard/keys")
+
+      assert response(conn, 200) =~ "API Keys"
```

Comment:
please run `mix format` to fix up indentation

Reply:
Thanks for your feedback @wojtekmach :)  I've updated the code.